src/Components/CreateEventForm.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import '../Styles/CreateEvent.css';
+import apiCalls from '../utilities/apiCalls';
+
+interface CreatedEvent {
+  id: number
+  date: string
+  category: string
+  start_time: string
+  end_time: string
+  vols_required: number
+  description: string
+}
+
+
+
+const CreateEventForm: React.FC = () => {
+
+const [category, setCategory] = useState<CreatedEvent["category"]>()
+const [date, setDate] = useState<CreatedEvent["date"]>()
```

Comment:
Reminder to add functionality to restrict date to current date and beyond.
Reminder to add functionality to restrict an end-time that starts prior to start-time